test/acceptance/replication/setup_test.go

```diff
@@ -28,8 +28,11 @@ func TestReplication(t *testing.T) {
 		t.Run("graphql search", graphqlSearch)
 		t.Run("multi-tenancy enabled", multiTenancyEnabled)
 	}
+}
+
+func TesAsyncReplication(t *testing.T) {
+	t.Setenv("TEST_WEAVIATE_IMAGE", "weaviate/test-server")
```

Comment:
I think that this should be removed, bc you are now binding `TEST_WEAVIATE_IMAGE` env var to only one possible value